ui/lib/core/addon/components/replication-dashboard.js

```diff
@@ -1,6 +1,20 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 import layout from '../templates/components/replication-dashboard';
 
 export default Component.extend({
   layout,
+  data: null,
+  dr: computed('data', function() {
+    let dr = this.data.dr;
+    if (!dr) {
+      return false;
+    }
+    return dr;
+  }),
+  isSyncing: computed('data', function() {
+    if (this.dr.state === 'merkle-sync' || this.dr.state === 'merkle-diff') {
```

Comment:
I wonder if instead of using magic strings here we could put these various states in one place and then reference them, so that if they changed in the future (or even just to see what they are) they'd be easier to track down. eg
`const MERKLE_STATES = {
sync: 'merkle-sync',
diff: 'merkle-diff',
// ....etc
}`

Reply:
Good idea.